covsirphy/analysis/scenario.py

```diff
@@ -37,6 +38,9 @@ def __init__(self, jhu_data, population_data, country, province=None, tau=None,
         # Linear regression modes for prediction of ODE parameters
         self._oxcgrt_data = None
         self._lm_dict = {}
+        # Default delay days
+        self.delay_days = jhu_data.recovery_period 
```

Comment:
Tests faild with this new line because of a bug of `ExampleData` class in cleaning/example_data.py (issue #549).
To fix this issue, please add `self._recovery_period = None` to `ExampleData` class.